src/test/Makefile

```diff
@@ -438,11 +438,19 @@ COMMON_FLAGS = \
 	-O0 \
 	-DUNIT_TEST \
 	-isystem $(GTEST_DIR)/inc \
-	-MMD -MP
+	-MMD -MP \
+	-Wno-c99-extensions \
+	-Wno-reorder
 
 CC_VERSION = $(shell $(CC) -dumpversion)
 CXX_VERSION = $(shell $(CXX) -dumpversion)
 
+# Betaflight supports clang 10. Revisit when new version arrive.
+CC_SUPPORTED_VERSION := $(shell expr `clang -dumpversion | cut -f1 -d.` \>= 10)
```

Comment:
Not sure how far back we should go, but I'd probably check for >= 6 as well.

Reply:
Still trying to figure this out...

Reply:
Without the use of extra libraries and no real OR function eventually have chosen a simple solution. I tried with fabrics etc. There could be better ways but tested and it only accept `clang` `6` to `10` otherwise presents the error and stops.

Reply:
💡 Now have to think about the case where the user has no clang installed at all